src/dotnet-scaffolding/dotnet-scaffold/Flow/Steps/ParameterDiscovery.cs

```diff
@@ -258,6 +259,22 @@ internal IList<string> GetProjectsFromSolutionFiles(List<string> solutionFiles,
             return projectPaths;
         }
 
+        internal List<string> AddUniqueProjects(List<string> baseList, List<string> projectPathsToAdd)
+        {
+            var baseProjectNames = baseList.Select(x => Path.GetFileName(x));
+            foreach(var projectPath in projectPathsToAdd)
+            {
+                var normalizedPath = StringUtil.NormalizePathSeparators(projectPath);
+                var projectName = Path.GetFileName(normalizedPath);
+                if (!baseProjectNames.Contains(projectName))
```

Comment:
We should be explicit about what Contains is going to do here. That probably means setting up BaseProjectNames as a List<string> with a specified comparer first, so you know it'll do the appropriate comparison. One question would be what is the appropriate comparison... technically linux/mac are case-sensitive and windows is case-insensitive.

Reply:
oh yeah totally forgot unix being case-sensitive. I think we should make this case-sensitive, thoughts?

Reply:
added comment, keeping it case-insensitive